superset-frontend/src/views/CRUD/welcome/Welcome.tsx

```diff
@@ -200,9 +197,14 @@ function Welcome({ user, addDangerToast }: WelcomeProps) {
   };
 
   useEffect(() => {
+    const defaultArr = ['2', '3'];
```

Comment:
Should we move this to the top, make it upper case and use it as a default value for `activeState` as well?

Reply:
was this after you had viewed data?

Reply:
Makes sense!